services/src/main/java/io/grpc/services/ChannelzProtoUtil.java

```diff
@@ -127,11 +131,42 @@ static ServerData toServerData(ServerStats stats) {
         .build();
   }
 
+  static Security toSecurity(Channelz.Security security) {
+    Preconditions.checkNotNull(security);
+    Preconditions.checkState(
+        security.tls != null ^ security.other != null,
+        "one of tls or othersecurity must be non null");
+    if (security.tls != null) {
+      Tls.Builder tlsBuilder
+          = Tls.newBuilder().setStandardName(security.tls.cipherSuiteStandardName);
+      if (security.tls.localCert != null) {
+        tlsBuilder.setLocalCertificate(ByteString.copyFrom(
+            security.tls.localCert.toString(),
```

Comment:
I'm surprised that this is an arbitrary textual representation, especially since `local_certificate` is `bytes`. Is this not supposed to be the binary form (DER)? That is available via `getEncoded()`. I would agree the .proto isn't all that specific and should probably be clarified (and that the gRFC is silent on the issue).

Reply:
I'll just use the `getEncoded` for now which returns ~600 bytes. toString may be a useful representation but we can add that later if needed.